tools/Versions/SetModuleVersion.ps1

```diff
@@ -50,7 +50,7 @@ function Invoke-BumpMinorOrPreReleaseVersion {
     }
     else {
         $version = [System.Version]("$($versionSegments[0])")
-        $newVersion = "$($version.Major).$($version.Minor + 1).$($version.Build)"
+        $newVersion = "$($version.Major).$($version.Minor + 1).0"
```

Comment:
Should the minor version always get incremented by 1 whenever this script is run? Minor version implies new features have been added.

Reply:
Yes, all SDKs treat new APIs as new features. The auto increment of minor version (this script) only happens when new APIs are added.